packages/migration-graph-shared/src/entities/package-graph.ts

```diff
@@ -111,9 +120,19 @@ export class PackageGraph {
           return;
         }
 
-        const relativePath = resolveRelative(baseDir, d.resolved);
+        const targetPath = resolveRelative(baseDir, d.resolved);
+        const packageName = this.package.packageName;
 
-        const dest = this.addNode({ key: relativePath, path: relativePath });
+        if (this.isFileExternalToPackage(targetPath)) {
+          console.warn(
```

Comment:
same as above. the user will never see this.

Reply:
I'd like to address this in a later PR, this issue will address it #603